components/generic_modal.tsx

```diff
@@ -23,6 +23,7 @@ type Props = {
     autoCloseOnCancelButton?: boolean;
     autoCloseOnConfirmButton?: boolean;
     enforceFocus?: boolean;
+    container?: React.ReactNode;
```

Comment:
`React.ReactNode | React.ReactNodeArray`

Reply:
does the container need to be an array of nodes? I would add it for the children prop but in this case i feel like its always going to be one element.